components/item-info.js

```diff
@@ -174,6 +178,18 @@ export default function ItemInfo ({
             <hr className='dropdown-divider' />
             <MuteSubDropdownItem item={item} sub={sub} />
           </>}
+        {canPin &&
+          <>
+            <hr className='dropdown-divider' />
+            <PinSubDropdownItem item={item} />
+          </>}
```

Comment:
Why did you decide to put this into a own section?

I first put it into the territory section (since only founders can pin) but when I added comment pinning, I moved it the "item section".

Reply:
Oh there are sections? lol

Reply:
Ah ok, nevermind. It's in the same section (between two `<hr className='dropdown-divider' />`) as `crosspost to nostr`. I think it makes sense now. I didn't read the code properly.